config/custom-environment-variables.json

```diff
@@ -13,10 +12,7 @@
   "maxLogLine": "MAX_LOG_LINE",
   "idamClient": "IDAM_CLIENT",
   "microservice": "MICROSERVICE",
-  "now": {
-    "__name": "NOW",
-    "__format": "json"
-  },
+  "now": "NOW",
```

Comment:
careful here, still need this fix in place

Reply:
Done @paddybasi